packages/liburcu/liburcu.spec

```diff
@@ -0,0 +1,65 @@
+Name: %{_cross_os}liburcu
+Version: 0.14.0
+Release: 1%{?dist}
+Summary: Library for userspace RCU
+License: LGPL-2.1-only AND GPL-2.0-or-later AND MIT
+URL: http://liburcu.org
+Source0: http://lttng.org/files/urcu/userspace-rcu-latest-0.14.tar.bz2
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Files for development using the library for userspace RCU
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n userspace-rcu-%{version}
+
+%build
+%cross_configure
+
+# "fix" rpath
+sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
```

Comment:
I'm curious why is this needed. Could you please elaborate on why this replacement is needed? Should we do what fedora does?

```spec
# Reinitialize libtool with the fedora version to remove Rpath
autoreconf -vif -W all,error
%configure --disable-static # You could include --disable-static and save yourself the find command below
```